src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -128,6 +129,12 @@ export class TxSender {
       }
     }
 
+    // If 1/? threshold and owner chooses to execute created tx immediately
```

Comment:
> 1/?

I doubt it's just for 1/? Safes.

Reply:
When else is it possible to create and execute a transaction immediately?

Reply:
It's true for creation, but isn't this onComplete also used for signing?
Oh, never mind. You're right. This code is after an early return that happens on sign.

Reply:
Yeah, it's a complicated logic flow. Maybe we should think about a better way of splitting the logic 🧐